internal/build/job/manager.go

```diff
@@ -15,46 +16,46 @@ import (
 	"github.com/kubernetes-sigs/kernel-module-management/internal/utils"
 )
 
-type jobManager struct {
+type podManager struct {
 	client    client.Client
 	maker     Maker
-	jobHelper utils.JobHelper
+	podHelper utils.PodHelper
 	registry  registry.Registry
 }
 
 func NewBuildManager(
 	client client.Client,
 	maker Maker,
-	jobHelper utils.JobHelper,
-	registry registry.Registry) *jobManager {
-	return &jobManager{
+	podHelper utils.PodHelper,
+	registry registry.Registry) *podManager {
+	return &podManager{
 		client:    client,
 		maker:     maker,
-		jobHelper: jobHelper,
+		podHelper: podHelper,
 		registry:  registry,
 	}
 }
 
-func (jbm *jobManager) GarbageCollect(ctx context.Context, modName, namespace string, owner metav1.Object) ([]string, error) {
-	jobs, err := jbm.jobHelper.GetModuleJobs(ctx, modName, namespace, utils.JobTypeBuild, owner)
+func (jbm *podManager) GarbageCollect(ctx context.Context, modName, namespace string, owner metav1.Object) ([]string, error) {
```

Comment:
```suggestion
func (pm *podManager) GarbageCollect(ctx context.Context, modName, namespace string, owner metav1.Object) ([]string, error) {
```
Please rename all receivers in this file.

Reply:
Thanks. Fixed.